catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -360,72 +353,22 @@ public UpdateResponse update(UpdateRequest updateRequest) throws IngestException
     }
 
     // map of old metacards to be populated
-    Map<Serializable, Metacard> idToMetacardMap = new HashMap<>();
-
     /* 1c. Populate list of old metacards */
```

Comment:
❓ Thanks for extracting methods and reducing the method size. Think we can remove the numbered comments (e.g., `/* 1c. Populate list of old metacards */`) now?